Gopkg.toml

```diff
@@ -10,7 +10,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
```

Comment:
`release/2.15` should be here, that's why you are having changes in vendor folder like start_pt_mongo_db_summary_action_parameters.go etc.

Reply:
but i need my branch with my changes

Reply:
you changed the base branch from `master` to `release/2.15`, but didn't pull new changes from `release/2.15`